minecraft_servers/zeenmc/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "zeenmc",
+    "nice_name": "ZeenMC Network",
+    "direct_ip": "play.zeenmc.es",
+    "server_wildcards": [
+      "%.zeenmc.com"
+    ],
+    "supported_languages": [
+      "es"
+    ],
+    "social": {
+      "web": "https://zeenmc.es",
+      "web_shop": "https://tienda.zeenmc.es",
+      "twitter": "zeenmc_es",
+      "discord": "https://discord.zeenmc.es",
```

Comment:
Custom links are reserved for partners only; use a normal discord.gg/[CODE] link instead